app/models/feedback.rb

```diff
@@ -14,6 +14,12 @@ class Feedback < ApplicationRecord
 
   default_scope -> { order created_at: :desc }
 
+  def self.authorized(user = Current.user)
+    return all if user&.role_admin?
+    return none unless user&.role_regional_admin?
+    where(issue_id: Issue.authorized.select(:id))
```

Comment:
```suggestion
    where issue_id: Issue.authorized(user)
```
Die Ergänzung der ID in der Projektion ist in den aktuellen Rails-Versionen nicht mehr notwendig. 
Dafür muss aber der `user` mit an die Einschränkung auf die Meldungen übergeben werden. Sonst funktionert eine Anfrage mit dem Nicht-Current-User, die gemäß API ja möglich ist nicht sauber, sondern wechselt plötzlich auf den Current-User!